srcs/juloo.keyboard2/KeyValue.java

```diff
@@ -156,8 +156,8 @@ private static void addEventKey(String name, String symbol, int event, int flags
   {
     addModifierKey("shift", "\uE808",
         FLAG_LOCK | FLAG_SHIFT | FLAG_KEY_FONT | FLAG_SMALLER_FONT);
-    addModifierKey("ctrl", "Ctrl", FLAG_CTRL | FLAG_SMALLER_FONT);
-    addModifierKey("alt", "Alt", FLAG_ALT | FLAG_SMALLER_FONT);
+    addModifierKey("ctrl", "Ctrl", FLAG_LOCK | FLAG_CTRL | FLAG_SMALLER_FONT);
+    addModifierKey("alt", "Alt", FLAG_LOCK | FLAG_ALT | FLAG_SMALLER_FONT);
```

Comment:
Modifiers can now be made lockable in the settings but the long press idea is nice.

Reply:
Forgot to ping here, long press have been implemented for every modifiers in 53b9afa, the double-tap locking is removed (except for Shift, where it can be enabled again in the settings).